main.qml

```diff
@@ -806,21 +806,27 @@ ApplicationWindow {
                     + ", fee: " + walletManager.displayAmount(transaction.fee));
 
             // here we show confirmation popup;
-            transactionConfirmationPopup.title = qsTr("Please confirm transaction:\n") + translationManager.emptyString;
-            transactionConfirmationPopup.text = "";
-            transactionConfirmationPopup.text += (address === "" ? "" : (qsTr("Address: ") + address));
-            transactionConfirmationPopup.text += (paymentId === "" ? "" : (qsTr("\nPayment ID: ") + paymentId));
-            transactionConfirmationPopup.text +=  qsTr("\n\nAmount: ") + walletManager.displayAmount(transaction.amount);
-            transactionConfirmationPopup.text +=  qsTr("\nFee: ") + walletManager.displayAmount(transaction.fee);
-            transactionConfirmationPopup.text +=  qsTr("\nRingsize: ") + (mixinCount + 1);
-            transactionConfirmationPopup.text +=  qsTr("\n\nNumber of transactions: ") + transaction.txCount
-            transactionConfirmationPopup.text +=  (transactionDescription === "" ? "" : (qsTr("\nDescription: ") + transactionDescription))
-            for (var i = 0; i < transaction.subaddrIndices.length; ++i){
-                transactionConfirmationPopup.text += qsTr("\nSpending address index: ") + transaction.subaddrIndices[i];
+            var transactionAmount = Number(walletManager.displayAmount(transaction.amount));
+            var transactionFee = Number(walletManager.displayAmount(transaction.fee));
+
+            function showFiatConversion(valueXMR) {
+                return (fiatApiConvertToFiat(valueXMR) === "0.00" ? "(<0.01 " + fiatApiCurrencySymbol() + ")" : "(~" + fiatApiConvertToFiat(valueXMR) + " " + fiatApiCurrencySymbol() + ")");
             }
 
+            transactionConfirmationPopup.dialogHeight = 425
+
+            transactionConfirmationPopup.title = qsTr("Confirm send") + translationManager.emptyString;
+            transactionConfirmationPopup.text =  "";
+            transactionConfirmationPopup.text += qsTr("From") + ":\n" + walletName + " (" + currentWallet.getSubaddressLabel(currentWallet.currentSubaddressAccount, 0) + ")";
+            transactionConfirmationPopup.text += "\n\n" + qsTr("To") + ":\n" + address;
+            transactionConfirmationPopup.text += "\n\n" + qsTr("Amount") + ":\n" + transactionAmount + " XMR " + (persistentSettings.fiatPriceEnabled ? showFiatConversion(transactionAmount) : "");
+            transactionConfirmationPopup.text += "\n\n" + qsTr("Fee") + ":\n" + transactionFee + " XMR " + (persistentSettings.fiatPriceEnabled ? showFiatConversion(transactionFee) : "");
+
             transactionConfirmationPopup.text += translationManager.emptyString;
             transactionConfirmationPopup.icon = StandardIcon.Question
+            transactionConfirmationPopup.cancelText = qsTr("Back") + translationManager.emptyString;
```

Comment:
Move it to `transactionConfirmationPopup` properties ->
https://github.com/monero-project/monero-gui/blob/35b7a5f6bb469583f1e78c9f35f154d312d99dc6/main.qml#L1399